python/src/main/java/org/apache/apex/malhar/stream/api/PythonApexStream.java

```diff
@@ -0,0 +1,32 @@
+package org.apache.apex.malhar.stream.api;
+
+/**
+ * Created by vikram on 7/6/17.
+ */
+public interface PythonApexStream<T> extends ApexStream<T>
+{
+
+  /**
+   * Add custom serialized Python Function along with options
+   * @param serializedFunction stores Serialized Function data
+   * @return new stream of type T
+   */
+  <STREAM extends ApexStream<T>> STREAM map_func(byte[] serializedFunction, Option... opts);
```

Comment:
can the function to be renamed as map and similarly other functions as flatmap / filter